### PR TITLE
Configurable dbus properties

### PR DESCRIPTION
**Pull request by jaor**
Alternative spotify clients such as spotifyd or mopidy can set
different dbus service names and also be accessible over the system
DBUS interface.  This patch makes both properties customisable.

### CODE REVIEW

File: spotify.el [hunk 1]
```diff
@@ -70,11 +88,16 @@
...
+
+(defun spotify--dbus () (if spotify-use-system-bus-p :system :session))
+
 (spotify-eval-only-dbus
   (defun spotify-dbus-call (interface method)
     "On INTERFACE call METHOD via D-Bus on the Spotify service."
-    (dbus-call-method-asynchronously :session
-                                     "org.mpris.MediaPlayer2.spotify"
+    (dbus-call-method-asynchronously (spotify--dbus)
+                                     (spofity--dbus-service-name)
```
Comment (remvee) [inline spotify.el:100]: Typo.  Causes `(void-function spofity--dbus-service-name)` on all player commands.

File: spotify.el [hunk 2]
```diff
@@ -70,11 +88,16 @@
   (unless (spotify-p-dbus)
     (eval `(quote (progn ,@body)))))
 
+(defun spotify--dbus-service-name ()
+  (format "org.mpris.MediaPlayer2.%s" spotify-service-name))
+
+(defun spotify--dbus () (if spotify-use-system-bus-p :system :session))
```
Comment (remvee) [inline spotify.el:94]: Yes, it probably sounds silly but I'd prefer `spotify--dbus-bus` or `spotify--dbus-bus-address`.

Also, put the `if` part on a new line, like you did for `spotify--dbus-service-name`.

**Comment by jaor** [comment]
sorry about the sloppiness of the previous patch.  i think i've
addressed your comments now, in a rebased version.  thanks!

**Comment by remvee** [comment]
Thanks!

**Pull request merged**